packages/connect-ui/src/lib/nango.ts

```diff
@@ -0,0 +1,6 @@
+import Nango from '@nangohq/frontend';
+
+export const nango = new Nango({
+    publicKey: import.meta.env.VITE_LOCAL_PUBLIC_KEY,
```

Comment:
tried to run the branch but getting a publicKey is not a uuid v4 error. How should I be passing this env var?

Reply:
in the PR desc there are some instructions, you should be able to test just by creating a `.env` in connect-ui, let me know